### DISCUSSION TITLE
Convert to only text document

### DISCUSSION THREAD

**Discussion by bleedingsaber** [akosbalasko/yarle#318] [Q&A] [answered]
Is it possible to strip html, only keep text, and no photo link?
Actually I only want text document from my evernote to Obsidian Vault.
Thanks

**Comment by akosbalasko** [comment, marked as answer]
Hi @bleedingsaber ,
Partially yes, if you don't mind that the output is in md format, which is plain text but with some pre-format characters within. However the photos are kept anyway, currently there's no option to exclude them.

**Reply by bleedingsaber** [reply]
Ok. Thanks for your kind reply. Your work is really helping. For me, I just find most photos in notes are not very necessary, they just make notes beautify but can not express ideas. I want my Vault keep simple clean and all full of key message. So I decide to use photos as little as I can. Actually use a lot of audio with media extent plugin. Anyway, thank you!
